.travis.yml

```diff
@@ -32,6 +32,11 @@ cache:
 # If inline scripts get too long, Travis surprisingly prints them twice.
 
 before_install:
+  # Travis pre-installs an old version of numpy. Uninstalling it
+  # manually has been known to solve some strange failures that we've
```

Comment:
Maybe make this slightly more specific, e.g. "Uninstall Travis's pre-installed numpy to avoid possibly strange behavior when upgrading in-place for tf-nightly's numpy dependency."

Reply:
Sure; will do.